components/logins/src/migrate_sqlcipher_db.rs

```diff
@@ -225,12 +415,22 @@ fn migrate_logins(
     let tx = conn.unchecked_transaction()?;
 
     for login in migration_plan.logins.values() {
-        // Migrate local login first
+        // Could not easily use an equality here due to the inner value
+        // But neccesary to ensure we log proper metrics
+        match &login.migration_op {
+            MigrationOp::Skip(err) => {
+                num_failed_insert += 1;
+                insert_errors.push(err.to_string());
+                continue;
```

Comment:
That enum variant is declared as `Skip(String)`, so I think `err` is already a string?